### PR TITLE
E2E tests

### PR DESCRIPTION
**Pull request by 7sne**
Closes <https://github.com/dethcrypto/dethtools/issues/65#issue-1299274802>

### CODE REVIEW

**Comment by LogvinovLeon** [comment]
After discussing with @a we were able to come up with one e2e test that would make sense. It would open an app, click through a couple of tools using them and then assert that there are no errors in the console. That way if ever react has a problem rendering our stuff, or smth else bad happens - it would be checked for by tests

**Comment by krzkaczor** [comment]
Please move e2e tests to a seprate job.

File: .eslintignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 /node_modules
 /public
 /styles
+.eslintrc.js
+test/e2e
```
Comment (krzkaczor) [inline .eslintignore:5]: why ignoring tests?

File: cypress/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (krzkaczor) [inline cypress/fixtures/example.json:1]: do we use this file?

File: cypress/support/commands.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+// This example commands.ts shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+
+export {};
```
Comment (krzkaczor) [inline cypress/support/commands.ts:12]: do we use this file?

File: cypress/support/e2e.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/e2e.ts is processed and
```
Comment (krzkaczor) [inline cypress/support/e2e.ts:2]: same

File: cypress/support/index.d.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      /**
+       * Custom command to select DOM element by data-cy attribute.
```
Comment (krzkaczor) [inline cypress/support/index.d.ts:5]: same

File: pages/index.page.tsx [hunk 6]
```diff
@@ -5,12 +5,10 @@ export default function Home(): ReactElement {
   return (
     <div>
       <Head>
-        <title>Create Next App</title>
+        <title>Deth tools</title>
         <meta name="description" content="Generated by create next app" />
```
Comment (krzkaczor) [inline pages/index.page.tsx:9]: what about this?

File: tsconfig.json [hunk 7]
```diff
@@ -17,7 +18,7 @@
     "baseUrl": "."
   },
   "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "**/*.test.ts", "./*.js"],
-  "exclude": ["node_modules"],
+  "exclude": ["node_modules", "test/e2e/**/*.cy.ts"],
```
Comment (krzkaczor) [inline tsconfig.json:21]: why excluding these files is required?

**Review by krzkaczor — CHANGES_REQUESTED** [review]

File: cypress/support/index.d.ts (hunk 5 above)
Comment (7sne) [reply cypress/support/index.d.ts:5]: Yes, as it's needed for the proper type infer of cypress fn and so on.

File: cypress/support/commands.ts (hunk 3 above)
Comment (7sne) [reply cypress/support/commands.ts:12]: same answer like above

File: cypress/support/e2e.ts (hunk 4 above)
Comment (7sne) [reply cypress/support/e2e.ts:2]: same answer like above

**Review by krzkaczor — APPROVED** [review]

**Pull request merged**